rpm/resalloc.spec.tpl

```diff
@@ -106,6 +106,22 @@ Requires(pre): /usr/sbin/useradd
 The %name-server package provides the resalloc server, and
 some tooling for resalloc administrators.
 
+%package webui
+Summary:    %sum - webui part
+
+Requires:   %default_python-%srcname = %version-%release
+%if %{with python3}
+Requires: python3-flask
+%else
+Requires: python-flask
+%endif
```

Comment:
Nit, I think we can build the resalloc-webui package only for python3.  No need to suport this on EL7.